components/cli/src/command/finalize.rs

```diff
@@ -0,0 +1,178 @@
+// Copyright 2019 The Exonum Team
+//
+// Licensed under the Apache License, Version 2.0 (the "License");
+// you may not use this file except in compliance with the License.
+// You may obtain a copy of the License at
+//
+//   http://www.apache.org/licenses/LICENSE-2.0
+//
+// Unless required by applicable law or agreed to in writing, software
+// distributed under the License is distributed on an "AS IS" BASIS,
+// WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+// See the License for the specific language governing permissions and
+// limitations under the License.
+
+//! Standard Exonum CLI command used to combine a secret and all the public parts of the
+//! node configuration in a single file.
+
+use exonum::blockchain::GenesisConfig;
+use exonum::node::{ConnectInfo, ConnectListConfig, NodeApiConfig, NodeConfig};
+use failure::{bail, format_err, Error};
+use serde::{Deserialize, Serialize};
+
+use std::collections::BTreeMap;
+use std::net::SocketAddr;
+use std::path::PathBuf;
+use structopt::StructOpt;
+
+use crate::command::{ExonumCommand, StandardResult};
+use crate::config::{CommonConfigTemplate, NodePrivateConfig, NodePublicConfig, SharedConfig};
+use crate::io::{load_config_file, save_config_file};
+
+/// Generate final node configuration using public configs
+/// of other nodes in the network.
+#[derive(StructOpt, Debug, Serialize, Deserialize)]
+#[structopt(rename_all = "kebab-case")]
+pub struct Finalize {
+    /// Path to a secret part of a node configuration.
+    pub secret_config_path: PathBuf,
+    /// Path to a node configuration file which will be created after
+    /// running this command.
+    pub output_config_path: PathBuf,
+    #[structopt(long, short = "p")]
+    /// List of paths to public parts of configuration of all the nodes
+    /// in the network.
```

Comment:
Comment docs between macro attribute and a field ? Looks a little bit unusual, doesn't it ?

Reply:
I experimented a bit and still can't decide which one looks better :D

Reply:
I'd prefer an order like: `comment docs, marco attribute, field`.